packages/react/src/components/layout/Popin/Popin.stories.tsx

```diff
@@ -1,26 +1,32 @@
 import React from "react";
 import { useArgs } from "@storybook/client-api";
+import { action } from "@storybook/addon-actions";
 
 import Popin, { PopinProps } from "./index";
 import Text from "../../asorted/Text";
+import Button from "../../cta/Button";
 import theme from "../../../styles/theme";
 
 export default {
-  title: "Layout/Drawer/Popin",
+  title: "Layout/Popin",
   component: Popin,
-  argTypes: {
-    children: {
-      type: "text",
-      description: "Any valid React node",
-      required: true,
-      control: {
-        type: "text",
+  parameters: {
+    docs: {
+      description: {
+        component: `This component is a customisable pop-in component.<br />The component exports by default the container that defines how the component should be animated during the mount/unmount process<br />In addition to that, 3 different sections of the component (header, body and footer) are attached to the Popin default component.<br /><br />You can access them by calling the \`Popin.Header\`, \`Popin.Body\` and \`Popin.Footer\` components. All of these components are optional.<br />- \`Popin.Header\` is accepting an onClose callback. The cross icon is only rendered if the onClose callback is provided<br />- \`Popin.Body\` is just a customisable scrollable Flex component configured to take all the available space<br />- \`Popin.Footer\` is a customisable Flex component<br /><br />📝 playground parameters aren't linked to the real component props. They are extra story parameters added to help us to play with the component.`,
+      },
+      source: {
+        code: `<Popin isOpen={isOpen}>\n\t<Popin.Header onClose={onBack} onClose={onBack}><Text variant="h5">A title</Text></Popin.Header>\n\t<Popin.Body><Text variant="paragraph">Some texts here</Text></Popin.Body>\n\t<Popin.Footer>\n\t\t<Button type="main" onClick={onClick}>\n\t\t\tNext\n\t\t</Button>\n\t</Popin.Footer>\n</Popin>`,
```

Comment:
Same here (not readable), also indentations with tabs (`\t`) look a bit too much IMO:

<img width="315" alt="Capture d’écran 2021-11-17 à 11 37 46" src="https://user-images.githubusercontent.com/86958797/142185147-b851f529-b597-4c72-b25f-febe95a799b4.png">